shared/shared.go

```diff
@@ -306,3 +317,23 @@ func (r *RootArgs) PrintMissingFlags(missingFlagNames []string) error {
 	}
 	return nil
 }
+
+func (r *RootArgs) GetProductsURL() string {
+	return fmt.Sprintf(productsURLFormat, r.RemoteServiceProxyURL)
```

Comment:
nit: This and below are really easy to cover with stand-alone tests. WDYT?

Reply:
Good point. Missed covering that.